src/Stratis.Bitcoin.Features.PoA/PoAMiner.cs

```diff
@@ -276,6 +283,28 @@ private Script GetScriptPubKeyFromWallet()
             return address.Pubkey;
         }
 
+        /// <summary>Adds OP_RETURN output to a coinbase transaction which contains encoded voting data.</summary>
+        /// <remarks>If there are no votes scheduled output will not be added.</remarks>
+        private void AddVotingData(BlockTemplate blockTemplate)
+        {
+            List<VotingData> scheduledVotes = this.votingManager.GetAndCleanScheduledVotes();
```

Comment:
This I am not clear about, can you put more then one vote in a block?

Reply:
Yes, you can vote for several things in one block

Reply:
The size of the op return is limited though

Reply:
It's a standard tx policy so miners can go beyond 80 bytes